src/static/css/2019.css

```diff
@@ -9,10 +9,10 @@
     local('Lato Italic'),
     local('Lato-Italic'),
     /* url(https://fonts.gstatic.com/s/lato/v16/S6u8w4BMUTPHjxsAUi-qNiXg7eU0.woff2) */
-    url('/static/fonts/Lato-Italic-ext.woff2')
-    format('woff2');
+    url('/static/fonts/Lato-Italic-ext.woff2') format('woff2');
```

Comment:
Seems like a lot of needless change in this file. Presume you're using a code formatter?

I'm not averse to any of the changes per se, but it does make it difficult to see the actual changes this PR is introducing. Is it possible to only include those?

Reply:
I see a lot of these changes are also causing our linter issues so definitely need to be reverted.

Reply:
I have installed linter as suggest but I don't know why that is not working, I am trying to resolve that, and soon make a commit.

Reply:
The linter just checks things meet our preferred style - it doesn't change them to make them meet that style so it's just an alerting system. you need to change them manually.

You must have some auto code formatter in your editor that's that's doing these changes?

Reply:
@bazzadp How do I fix these?

Reply:
I think easiest is to take the old version [from here](https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/src/static/css/2019.css) and then apply you changes again (presume they are just the ones at the bottom?).

But be careful your code editor doesn't reformat them again. I presume you have some plugin that's doing this? If so might need to figure out what is doing that.

Do a `git diff` before committing that update anyway to make sure only the changes you intended to include are included and everything else is reverted back.